src/app/src/main/kotlin/org/vaccineimpact/api/app/SequenceExtensions.kt

```diff
@@ -10,7 +10,7 @@ import org.vaccineimpact.api.models.BurdenEstimateWithRunId
 // Note that the exception will originate from whatever part of the program is currently
 // asking for another element from the sequence, not from the caller of
 // this function.
-fun Sequence<BurdenEstimateWithRunId>.validate(expectedRows: HashMap<String, HashMap<Int, HashMap<Int, Boolean>>>
+fun Sequence<BurdenEstimateWithRunId>.validate(expectedRows: HashMap<String, HashMap<Short, HashMap<Short, Boolean>>>
```

Comment:
In the `expectations` table in the database ages and years are `SMALLINT`s, so I've moved to using `Short` everywhere in this code.